ui/lib/core/addon/components/search-select-with-modal.hbs

```diff
@@ -0,0 +1,107 @@
+<div {{did-insert this.fetchOptions}} ...attributes data-test-search-select-with-modal>
+  {{#if this.shouldUseFallback}}
+    {{component
+      @fallbackComponent
+      label=(or @label @subLabel)
+      subText=@subText
+      onChange=@onChange
+      inputValue=@inputValue
+      helpText=@helpText
+      placeHolder=@placeHolder
+      id=@id
+    }}
+  {{else}}
+    {{#if @label}}
+      <label for={{or @id @label}} class="is-label" data-test-field-label>
+        {{capitalize @label}}
+        name
```

Comment:
Is this `name` intentional?

Reply:
Yes - since @label was passed in to generate the `Create new <label>` text for the dropdown, I wanted @label to be more flexible, but that's not very reusable if the Search select is always referring to a `name` 

So changed the create text to use the id!